src/main/java/io/aiven/kafka/auth/json/AclPermissionType.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 Aiven Oy https://aiven.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.auth.json;
+
+public enum AclPermissionType {
+    ALLOW((byte) 1),
+    DENY((byte) 0);
+
+    private final byte code;
```

Comment:
Do we need this `code`? Seems that it should work just fine without it, in JSON it's encoded as string anyway.